lvgl_tft/Kconfig

```diff
@@ -469,14 +469,15 @@ menu "LVGL TFT Display controller"
 
     choice
         prompt "TFT SPI Bus." if LV_TFT_DISPLAY_PROTOCOL_SPI
-        default LV_TFT_DISPLAY_SPI2_HOST
-        help
-              Select the SPI Bus the TFT Display is attached to.
+        default LV_TFT_DISPLAY_SPI2_HOST if LV_PREDEFINED_DISPLAY_TTGO && !IDF_TARGET_ESP32S2
```

Comment:
`LV_PREDEFINED_DISPLAY_TTGO && !IDF_TARGET_ESP32S2` also should always be true if LV_PREDEFINED_DISPLAY_TTGO is picked.

Reply:
How can the user set `IDF_TARGET_ESP32S2`, afaik is with `idf.py set-target`

Reply:
Yeah, I also think `&& !IDF_TARGET_ESP32S2` part should be removed.